### DISCUSSION TITLE
Multiple logins detected - Monitor Zwift Login is being used by more than one application

### DISCUSSION THREAD

**Discussion by bentjeppesen** [SauceLLC/sauce4zwift#109] [General]
Using version 1.1.2 I have started getting an error saying that my Monitor Zwift Login is being used by more than one application. Error typically or maybe only appears on longer rides (after an hour). My monitor zwift login is to my knowledge only being used by sauce4zwift. After this msgbox, sauce4zwift stops. Anyone else having this problem?

![Udklip](https://github.com/SauceLLC/sauce4zwift/assets/120282113/a4239e2b-1689-4592-9c9b-8d4c51566958)
[sauce.log.zip](https://github.com/SauceLLC/sauce4zwift/files/14214183/sauce.log.zip)

**Comment by mayfield** [comment]
Should be fixed in 1.1.3-alpha See #106
